lib/encoding-status-view.js

```diff
@@ -64,6 +68,11 @@ export default class EncodingStatusView {
         this.encodingLink.textContent = encodingLabel.status
         this.encodingLink.dataset.encoding = editorEncoding
         this.element.style.display = ''
+
+        if(this.tooltip) {
+          this.tooltip.dispose()
+        }
+        this.tooltip = atom.tooltips.add(this.encodingLink, {title: 'File displayed using ' + encodingLabel.list + ' encoding'})
```

Comment:
I wonder if 'displayed' is the right word. It's parsed with the encoding so if you change encoding and save it you can lose information.  Displayed sounds like it's completely passive.  Also consider using string interpolation.